Source/ARTCrypto.m

```diff
@@ -255,7 +255,7 @@ - (ARTStatus *)decrypt:(NSData *)ciphertext output:(NSData *__autoreleasing *)ou
 - (NSString *)cipherName {
     NSString *algo = nil;
     switch (self.algorithm) {
-        case kCCAlgorithmAES128:
+        case kCCAlgorithmAES:
             algo = @"aes-128";
```

Comment:
Can we also update the cipherName to be consistent? e.g. `algo = @"aes"`?

Reply:
8364a0e

Reply:
@ben-xD please resolve this conversation if everything is ok. Thanks